Content.Shared/Access/Components/IdCardConsoleComponent.cs

```diff
@@ -74,6 +74,7 @@ public WriteToTargetIdMessage(string fullName, string jobTitle, List<ProtoId<Acc
         "Mime", // DeltaV - Add Mime access
         "Musician", // DeltaV - Add Musician access
         "Paramedic", // DeltaV - Add Paramedic access
+        "Pharmacist", // DeltaV - Add Pharmacist access
```

Comment:
The Pharmacist and the Psychologist have quite a lot of overlap, and I think they should be kept under the same access. Otherwise, for the 90% of stations that will not have them, this will be a button that does not do much taking up space in the console.

I might suggest something like "Psychiatry" as a shared access, renaming the existing Psychologist access and dropping Pharmacist entirely.

Reply:
I would argue that Zookeeper has its own access.

> `89 90 "Zookeeper",  // DeltaV - Add Zookeeper access`